schema.py

```diff
@@ -0,0 +1,44 @@
+from graphene_sqlalchemy import SQLAlchemyObjectType, SQLAlchemyConnectionField
+from graphene import relay, Schema, ObjectType, Mutation, String, Field, Boolean
+from models import User, Story, db_session
+
+
+class UserType(SQLAlchemyObjectType):
+    class Meta:
+        model = User
+        interfaces = (relay.Node,)
+
+class CreateUser(Mutation):
+  user = Field(UserType)
+
+  class Arguments:
+    first_name = String(required=True)
+    last_name = String(required=True)
+    username = String(required=True)
+    password = String(required=True)
+    email = String(required=True)
+  ok = Boolean()
+
+  def mutate(self, info, first_name, last_name, username, password, email):
```

Comment:
Works fine, but this mutate class will restrict you to supply all user fields including those that should be optional, consider using kwargs and only saving the fields supplied instead of requiring al fields